packages/apollo/src/commands/client/extract.ts

```diff
@@ -30,7 +37,9 @@ export default class ClientExtract extends ClientCommand {
       {
         title: "Extracting operations from project",
         task: async ctx => {
-          ctx.operations = getOperationManifestFromProject(this.project);
+          ctx.operations = getOperationManifestFromProject(this.project, {
+            stripLiterals: flags.stripLiterals
```

Comment:
I would also suggest renaming this to more accurately describe its scope:
```suggestion
            preserveStringAndNumericLiterals: flags.preserveStringAndNumericLiterals
```